src/infrastructure/jira/schemas.ts

```diff
@@ -16,3 +23,41 @@ export const ATTACHED_DESIGN_URL_V2_VALUE_SCHEMA: JSONSchemaTypeWithId<
 		required: ['url', 'name'],
 	},
 };
+
+export const NODE_DEV_STATUS_SCHEMA: JSONSchemaType<NodeDevStatus> = {
```

Comment:
**issue:** @stephbrook11 , please, move newly added schemas to `src/infrastructure/figma/figma-client/schemas.ts` as they belong to the Figma API contract.